src/oscar/templatetags/currency_filters.py

```diff
@@ -10,7 +10,7 @@
 
 
 @register.filter(name='currency')
-def currency(value, currency=None):
+def currency(value, currency=settings.OSCAR_DEFAULT_CURRENCY):
```

Comment:
I'd prefer not to specify the default here, because Python evaluates defaults at runtime, meaning overriding settings becomes tricky. Instead it's better to do set this inside the function if `currency` is `None`.